splat-radio.sh

```diff
@@ -36,8 +46,78 @@ if [ ! -x `which optipng` ]; then
 	exit 1
 fi
 
-# no more variables to edit below here
 
+function helptext {
+cat <<EOF
+
+Usage: $0 -c CONFIGFILE [-b]
+
+-c CONFIGFILE
+        The file with your configuration, see README.md
+        
```

Comment:
Why should we have these particular options on the command line and not the config file?  I'm not sure of the use case.